webview-ui/src/components/ChatPage/AgentFileReview.tsx

```
@@ -0,0 +1,395 @@
+  React.useEffect(() => {
+    if (pendingFiles.length === 0) {
+      setBatchOperationInProgress(false);
+    }
+  }, [pendingFiles.length, setBatchOperationInProgress]);
+
+  // Auto-expand the first file when the review appears
+  React.useEffect(() => {
+    if (pendingFiles.length > 0 && expandedTokens.size === 0) {
+      setExpandedTokens(new Set([pendingFiles[0].messageToken]));
+    }
+  }, [pendingFiles.length > 0]); // eslint-disable-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)